Source/Core/DolphinWX/Input/InputConfigDiag.h

```diff
@@ -23,10 +23,10 @@
 #include <wx/spinctrl.h>
 #include <wx/timer.h>
 
-#include "InputCommon/ControlReference/ControlReference.h"
-#include "InputCommon/ControllerEmu/ControllerEmu.h"
+#include "InputCommon/ControllerEmu/ControlGroup/ControlGroup.h"
```

Comment:
Currently no, because `NumericSetting` is still a nested class within ControlGroup. However this will be changed in a followup.